src/convertToObject.js

```diff
@@ -17,19 +17,19 @@ function convertToObject(sourceString) {
   const result = {};
   const arrOfProperties = sourceString.split(';');
 
-  for (const index in arrOfProperties) {
-    let elem = arrOfProperties[index];
+  for (let index = 0; index < arrOfProperties.length; index++) {
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead.
check it for whole code

Reply:
I have talked to Misha exactly about this itteration loop. So it done according his advices.